tasks/enable_restart_vpn.yml

```diff
@@ -9,10 +9,6 @@
   loop: "{{ __vpn_services }}"
 
 - name: Wait for ssh connection to return
-  wait_for:
-    host: "{{ ansible_default_ipv4.address |\
-          d(ansible_default_ipv6.address) }}"
-    port: 22
+  wait_for_connection:
     delay: 3
-    state: started
-  delegate_to: localhost
```

Comment:
No need this any more? Is wait_for_connection supposed to be executed on the managed hosts?

Reply:
modules like `ping`, `wait_for`, etc. are executed on the controller host, unless they are looking for some file or service to be running on the managed host, but even then, they run on the controller host, waiting until they can ssh to the managed host